examples/java/spring-boot/src/test/resources/application.properties

```
@@ -0,0 +1 @@
+fixedRate.in.milliseconds = 500
```

There's a recommendation by Spring (can't remember which docs it's from, so sorry for no link!) that we should use kebab-case for naming:
```suggestion
fixed-rate.in.milliseconds = 500
```